src/site/content/en/index.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: homepage
+description: |
+  Get the web's modern capabilities on your own sites and apps with useful
+  guidance and analysis from web.dev.
+date: 2018-11-05
+i18n:
+  banner_blm: |
```

Comment:
If we think we're going to include HTML in a tag, can we suffix it with `_html`, so we know that it should be passed through `| safe`?

Reply:
+1